src/components/TimelineComponent.tsx

```diff
@@ -338,6 +357,18 @@ export default class TimelineComponent
     });
   }
 
+  /**
+   * Stores the zoom object for programmatically adjusting the zoom
+   */
+  private zoom: any;
+  /**
+   * Stores the svg for appending elements
```

Comment:
Be more explicit for the noobs here, what is an svg? What is it's scope of responsibility?